app/javascript/components/provider-dashboard-charts/events-pie-chart/index.js

```diff
@@ -0,0 +1,73 @@
+import React, { useEffect, useState } from 'react';
```

Comment:
The file name indicates that this is a pie chart but the screenshot in the description looks like a bar chart

Reply:
Correct , I fixed it 
(the first concept was a pie chart)